website/docs/telemetry.md

```diff
@@ -35,23 +35,30 @@ Argument values, data field names, successful responses, or error messages are n
 ## Version notification
 
 When a FerretDB update is available, the telemetry service sends periodic reports containing information about the latest FerretDB version.
-
-This information is logged in the server logs, `startupWarnings`, and `serverStatus` command output.
```

Comment:
`serverStatus` never logged version update information, only the current telemetry state